Makefile

```diff
@@ -1,6 +1,6 @@
 all:
 	@echo "Making deviceconsole..."
-	@$(CC) -O3 main.c tail.c -o deviceconsole -F/System/Library/PrivateFrameworks/ -framework MobileDevice -framework CoreFoundation
+	@$(CC) -O3 main.c tail.c -o deviceconsole -F./f/ -framework MobileDevice -framework CoreFoundation
```

Comment:
This is not a default folder. `/System/Library/PrivateFrameworks/` should work though

Reply:
because **xcode9** can't link to `/System/Library/PrivateFrameworks/`,I had built it and get a error 
```
ld: framework not found MobileDevice
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
so ,I use a soft-link to fix this.
do you have test on Xcode 9?